publish/stack-java-openliberty/stack-java-openliberty.md

```diff
@@ -196,6 +196,8 @@ Successfully initialized Appsody project with the java-openliberty stack and the
 
 **Note:** Some lines (..) are removed for clarity.
 
+**Note:** In the latest release, the image for `java-openliberty` is in the `/appsody` folder instead of the `/kabanero` folder. To prevent confusion, review the stack hub yaml file for the appropriate path to use in the `appsody-config.yaml` file.
```

Comment:
I initialized an OL stack and the changes you made to the system output are spot on. However, I'm not sure that a **Note** is appropriate for this guide. Since the guide is for users who want to learn how to develop a microservice from the OL stack, this information is unlikely to be relevant here. If we do want to put a note somewhere, I think we'll need to revise the wording. 

.... the image for `java-openliberty` is pulled from `docker.io/appsody` instead of `docker.io/kabanero`....

Actually the whole stack comes from Appsody instead of Kabanero.

Reply:
Suggestion added to the issue